app/styles/crate.scss

```diff
@@ -466,7 +469,6 @@
     .small { margin-left: 20px; display: inline-block; }
     a.arrow {
         display: inline-block;
-        float: right;
```

Comment:
Removing this does change the semantics – the arrow won't be a float anymore, and it's not moved to the right side of the containing element anymore. It does not appear to make a difference on modern browsers because the containing element has `justify-content: space-between;`, which also results in moving the arrow to the right, but I'd be careful and just leave this in.

Reply:
Fixed.